src/main/java/bio/terra/service/filedata/DrsIdService.java

```diff
@@ -43,23 +43,27 @@ private DrsId fromParts(String snapshotId, String fsObjectId) {
         .build();
   }
 
-  public DrsId fromUri(String drsuri) {
+  public static DrsId fromUri(String drsuri) {
```

Comment:
should probably be `drsUri`
```suggestion
  public static DrsId fromUri(String drsUri) {
```

Reply:
Good point